hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/CalciteSqlOptimizer.java

```diff
@@ -804,8 +805,7 @@ private PhysicalRel optimize(
             logger.fine("After physical opt:\n" + RelOptUtil.toString(physicalRel));
         }
 
-        PhysicalRel finalPhysicalRel = physicalRel;
-        queryPlanListeners.forEach(l -> l.onQueryPlanBuilt(finalPhysicalRel));
+        // TODO[sasha]: capture final physical rel for listeners here.
```

Comment:
?

Reply:
It's for the future optimized-plan-listeners, I marked a place for myself where exactly locate it